pkg/functions/function_security.go

```diff
@@ -0,0 +1,8 @@
+package functions
+
+type PodSecurityContext struct {
```

Comment:
We should confirm with the Serving team, but this may need to be renamed `SecurityContext` and set at the Container level in the Knative Deployer in order to clear up the validation webhook error.